### ISSUE TITLE
EAP server adapter (7.3) is losing track of its state

### ISSUE THREAD

**Issue by odockal** [redhat-developer/intellij-rsp#34]
[kind/bug]

## What versions of software are you using?
- Operating System: Fedora 32
- IntelliJ version: 2018-03!
- Intellij-rsp extension version: from master

## What did you run exactly?
Downloaded EAP, started EAP (server started), Showed in browser -> it showed correct welcome EAP page, then Server state went to stop. Page and server is still running though...

**Comment by robstryker** [comment]
I'm not able to replicate this.

**Comment by robstryker** [comment]
@odockal  can you still replicate? I got nothing here

**Comment by odockal** [comment]
@robstryker hm, I could not. Let's close and consider it a glitch unless reported again...

**Comment by odockal** [comment]
Wau.... I have encounter bit same but also a bit different behavior. Again with eap 7.3. I downloaded the server and started it. Log says it is started, I also added deployment. But its state is keeping `STARTING`, so the option I can do with the server are limited, ie. I cannot show in browser. I played around a while (terminated the server, started again, tried to edit server, etc.) and all of sudden, puf, server is gone.
